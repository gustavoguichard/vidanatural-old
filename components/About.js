import { memo } from 'react'
import Link from 'next/link'
import ReactMarkdown from 'react-markdown'
import { Button, Columns, Content } from 'react-bulma-components'
import Logo from 'components/Logo'
import PageBreadCrumb from 'components/menu/PageBreadCrumb'
import ImageContainer from 'components/ImageContainer'
import { useMedia } from 'utils/hooks'

import content from 'content/sobre'
import 'styles/about.scss'

export default memo(() => {
  const isDesktop = useMedia('desktop')
  return (
    <Columns id="sobre" className="about-section is-fullheight" gapless>
      <Columns.Column
        key="design"
        className="about-header"
        style={{ display: 'flex' }}
      >
        <ImageContainer
          src="/static/about-bg.jpg"
          key="about-bg"
          className="about-bg"
          contentClass="about-bg-content"
          fixed
        >
          <Logo clickable key="logo" />
          {isDesktop && <PageBreadCrumb key="breadcrumb" title="sobre" />}
        </ImageContainer>
      </Columns.Column>
      <Columns.Column className="content-wrapper">
        <div className="banner-content-wrapper">
          <Content key="content" className="black-content banner-content">
            <h2 className="title is-4">Sobre a VN</h2>
            <ReactMarkdown className="md-content" source={content} />
          </Content>
          <p className="banner-content">
            <Link href="sobre">
              <Button color="light" className="is-large" rounded outlined>
                Quero saber mais
              </Button>
            </Link>
          </p>
        </div>
      </Columns.Column>
    </Columns>
  )
})
