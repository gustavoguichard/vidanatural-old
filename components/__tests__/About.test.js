import { render } from '@testing-library/react'
import About from 'components/About'

it('renders correctly', () => {
  const { container } = render(<About />)
  expect(container).toMatchSnapshot()
})
