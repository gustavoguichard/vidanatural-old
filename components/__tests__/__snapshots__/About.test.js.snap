// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="about-section columns is-gapless is-multiline"
    id="sobre"
  >
    <div
      class="about-header column"
      style="display: flex;"
    >
      <div
        class="image-container about-bg"
      >
        <div
          class="content-above about-bg-content"
        >
          <div
            class="logo"
            style="display: block; background: url(/static/logo-white.png) no-repeat center; background-size: contain; height: 117px; width: 130px; overflow: hidden; text-indent: -999em; cursor: pointer;"
          >
            <h1>
              Vida Natural
            </h1>
          </div>
        </div>
      </div>
    </div>
    <div
      class="content-wrapper column"
    >
      <div
        class="content black-content"
      >
        <h2
          class="title is-4"
        >
          Sobre
        </h2>
        <p>
          Consciêcia talvez seja a palavra mais importante no nosso vocabulário. E, com o tempo, você vai perceber que ela está presente nas nossas iniciativas e, também, na forma como nos relacionamos com as pessoas, produzimos e distribuímos os nossos cosméticos.
        </p>
        <p>
          Em breve, você conhecerá muito sobre nós, inclusive o nosso processo de produção. Tudo para motivar você a dizer
           
          <strong>
            eu uso
          </strong>
           cosmética consciente e nos auxiliar a completar a grafia do nosso posicionamento.
        </p>
      </div>
    </div>
  </div>
</div>
`;
